modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/etypefeatureprotocol/v0.1.1.json

```diff
@@ -7,7 +7,7 @@
   ],
   "@type": "nxv:Schema",
   "imports": [
-    "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1"
+    "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.2"
```

Comment:
Another schema version should created with the changes.

Reply:
Done